interface/src/app/help/help.tpl.html

```diff
@@ -88,12 +88,12 @@ <h3><u>Signatures</u></h3>
   <div class="lead" id="High Weight Gene">
     <h3><u>High Weight Gene</u></h3>
```

Comment:
How about:

> The ADAGE model is made up of features (also known as nodes). For each feature, every gene has a positive or negative weight. A signature is composed of the genes on one side (positive or negative) of a feature. The genes that are termed high weight are those that are 2.5 or more standard deviations from the mean gene weight. These high weight genes are used to characterize each signature because their expression levels have the largest influence on the activity of the signature.

Reply:
@casey, do you have any comments on the two paragraphs that describe `Nodes` (lines 61-66) and `Signatures` (lines 72-84) in `help.tpl.html`?